xchange-core/src/main/java/org/knowm/xchange/derivative/FuturesContract.java

```diff
@@ -38,7 +38,7 @@ public FuturesContract(CurrencyPair currencyPair, Date expireDate) {
 
   @JsonCreator
   public FuturesContract(final String symbol) {
-    String[] parts = symbol.split("/");
+    String[] parts = symbol.split("-");
     if (parts.length < 3) {
```

Comment:
May be I missed the logic of moving from the current standard format of slash to hyphens i.e. from `BTC/USDT/210924` to `BTC-USDT-210924` just this will break the api for a lot of people. If hyphens are required, they should be done in the individual exchange adapters.

Reply:
Regarding slash-hyphen change: I haven't really found proper usage of FuturesContract and the exact place that the change will affect. Before the PR I checked Deribit and Okex: both use hyphens. I assume slash is some kind of inheritance from currency pair, which will only look confusing for futures and options.

Bottom line: if you say it is already in use, I agree then, we keep slashes, I'll fix my PR.

Reply:
Reverted to slashes for futures: BTC/USD/210430, using slashes for options: BTC/USD/210730/40000/C